src/components/shared/newsletter/newsletter.jsx

```diff
@@ -0,0 +1,27 @@
+import RightArrow from '../svg/right-arrow/right-arrow';
+import styles from './newsletter.module.scss';
+
+export default function Newsletter() {
+  return (
+    <div className={styles.container}>
+      <form className={styles.form}>
+        <p className={styles.title}>NEWSLETTER</p>
+        <p className={styles.signUp}>Sign up for updates</p>
+        <p className={styles.description}>
+          Sign up with your email address to receive news and updates on my
+          latest exhibitions & works.
+        </p>
+        <div className={styles.row}>
+          <input
```

Comment:
type should be email. Also we can use the new email form element you built. No need to refactor it now but in the future once the other form elements are complete we can replace this one with that one so that it also has the validations in place, those are great.